lua/weapons/weapon_golfclub/shared.lua

```diff
@@ -211,268 +208,262 @@ function SWEP:SetupDataTables()
-	
-	local dist = math.Clamp(p1:Distance(p2),0,200)
-	
-	if(medist < 150 and p1:Distance(p2) > 3)then
-	can = true
-	local tarvec = p1
-	tarvec.z = ball:GetPos().z
-	local ballvec = ball:GetPos()
-	ball:StartShot()
-	ball:GetPhysicsObject():ApplyForceCenter((tarvec - ballvec)* 100)
-	local strk = self:GetCurrentStroke()
-	self:SetCurrentStroke(strk+1)
-	else
-	can = false
-	end
-	
-	
-	
-	
-	
-	end
-	
-	
-	
 		if(can)then
 		
 			self:SetShootStage(newstage)
 		
 		end
 	end
-self:SetNextPrimaryFire(CurTime()+0.125)
-self:SetNextSecondaryFire(CurTime()+0.125)
-return true
+	self:SetNextPrimaryFire(CurTime()+0.125)
+	self:SetNextSecondaryFire(CurTime()+0.125)
+	return true
 end
+
 function SWEP:FinishStroke(hole)
 	local joke = (hole=="Srsly2hard4u")
-local strk = self:GetCurrentStroke()
-if(hole)then
-local endm = strk.." strokes!"
-
-local snd = "hl1/fvox/bell.wav"
-if(strk == 1)then
-endm = "a Hole-in-One!"
-//snd = "skull-trumpet.wav"
-end
-self:GetOwner():EmitSound(snd)
-if joke then
-	local vd = self.Owner:GetPos()
-	timer.Simple(0.4, function() sound.Play("golfmines/amazing.wav", vd, 70, 100, 1) end)
-	self.CantPlaceBall = true
-	timer.Simple(7,function() if IsValid(self) then self.CantPlaceBall=false end end)
-end
-local record = 10
-if SERVER then
-	record = tonumber(self:GetOwner():GetPData("golf_"..hole, 10))
-	if strk<record then
-		self:GetOwner():SetPData("golf_"..hole, tostring(strk))
-	end
-end
-if !joke then GolfNotify("[yellow]"..self:GetOwner():Nick() .." completed hole '"..hole.. "' with "..endm) end
-if SERVER then
-	local iwonsomething = false
-	if strk<record then
-		local strokes = " strokes"
-		if strk==1 then strokes = " stroke" end
-		self:GetOwner():PrintMessage( HUD_PRINTTALK, "[red]You beat your record! New record: [orange]"..tostring(strk)..strokes )
-		local t = GolfPrizeTargets[hole]
-		if t then
-			for k,v in pairs(t) do
-				if record > v[1] and strk <= v[1] then
-					if self:GetOwner().familyshared then
-						self:GetOwner():PrintMessage( HUD_PRINTTALK, "[red]Sorry, you're not eligible for this prize ;hahaha;" )
-					else
-						self:GetOwner():PrintMessage( HUD_PRINTTALK, "[red]Congratulations! You won [gold]"..tostring(v[2]).." points!" )
-						self:GetOwner():PS_GivePoints(v[2])
-					end
-					iwonsomething = true
+	local strk = self:GetCurrentStroke()
+	if(hole)then
+		local endm = strk.." strokes!"
+		
+		local snd = "hl1/fvox/bell.wav"
+		if(strk == 1)then
+			endm = "a Hole-in-One!"
+			//snd = "skull-trumpet.wav"
+		end
+		self:GetOwner():EmitSound(snd)
+		if joke then
+			local vd = self.Owner:GetPos()
+			timer.Simple(0.4, function() sound.Play("golfmines/amazing.wav", vd, 70, 100, 1) end)
+			self.CantPlaceBall = true
+			timer.Simple(7,function() if IsValid(self) then self.CantPlaceBall=false end end)
+		end
+		local record = 10
+		if SERVER then
+			record = tonumber(self:GetOwner():GetPData("golf_"..hole, 10))
+			monthlyrecord = tonumber(self:GetOwner():GetPData("monthly_golf_"..hole, 10))

```

Comment:
Can you tell the player something about what the player's all-time record is vs their monthly record if the two differ?